bedrock/mozorg/templates/mozorg/home/includes/nav.html

```diff
@@ -0,0 +1,38 @@
+<header class="section masthead" id="masthead">
+  <div class="content">
+    <h1 class="masthead-logo">
```

Comment:
Hmm, as this is going to be a shared header (which totally makes sense 👍), this probably doesn't want to be an `h1` on every page, as multiple `h1`'s are no longer considered valid. The exception is probably the home page, which already uses an `h2` for the main content heading and would benefit from an `h1` here.